core/src/main/java/com/backblaze/b2/client/B2LargeFileStorer.java

```diff
@@ -42,6 +48,13 @@
      */
     private final List<B2PartStorer> partStorers;
 
+    /**
+     * Stores where each part will begin in the finished large file.
+     * Will be set to B2UploadProgress.UNKNOWN_PART_START_BYTE for
+     * parts that come after a copied part.
+     */
+    private final List<Long> startingBytePositions;
```

Comment:
It's possible for them all to be UNKNOWN_PART_START_BYTE.  What's the benefit of having these if you can't count on them?

Reply:
ab and I discussed having the `B2LargeFileStorer` replace the `B2LargeFileUploader`. Since we didn't want to lose that functionality, we provide a best-effort solution. For purely uploaded files, the new behavior should be identical to the old.